src/management/research.c

```diff
@@ -186,7 +186,7 @@ void research_finish_item(sint32 entryIndex)
 		gResearchedTrackTypesA[base_ride_type] = (RideTypePossibleTrackConfigurations[base_ride_type]         ) & 0xFFFFFFFFULL;
 		gResearchedTrackTypesB[base_ride_type] = (RideTypePossibleTrackConfigurations[base_ride_type] >> 32ULL) & 0xFFFFFFFFULL;
 		if (RideData4[base_ride_type].flags & RIDE_TYPE_FLAG4_3) {
-			ebx = RCT2_GLOBAL(0x0097D4F5 + (base_ride_type * 8), uint8);
+			ebx = RideData4[base_ride_type].var_3;
```

Comment:
This code might be wrong, it looks like var_3 represents a track set:
https://github.com/OpenRCT2/OpenRCT2/blob/develop/src/ride/track.h#L79

but I think what its indexing into `RideTypePossibleTrackConfigurations` should probably be `base_ride_type `.